README.md

```diff
@@ -2,727 +2,369 @@
-
-* Windows 7.
-* Linux.
-* Mac OS X (Darwin).
-* Raspbian Jessie Lite.
-
-# Rationale
-
-When connecting more than one mbed-enabled device to the host computer, it takes time to manually check the platforms' binds:
-
-* Mount point (MSD / disk).
-* Virtual serial port (CDC).
-* mbed's TargetID and generic platform name.
-
-```mbed-ls``` provides these points of information for all connected boards at once in a simple console (terminal) output.
+# mbed-ls
+
+`mbed-ls` is a Python (2 and 3) module that detects and lists Mbed Enabled devices connected to the host computer. It is delivered as a redistributable Python module (package) and command-line tool. It works on all major operating systems (Windows, Linux and Mac OS X).
+
+It provides the following information for all connected boards in a simple console (terminal) output:
+
+- Mbed OS platform name.
+- Mount point (MSD or disk).
+- Serial port.
 
 # Installation
 
 ## Installation from PyPI (Python Package Index)
 
-mbed-ls module is redistributed via PyPI. We recommend you use the [application pip](https://pip.pypa.io/en/latest/installing.html#install-pip).
+To install `mbed-ls` from [PyPI](https://pypi.python.org/pypi/mbed-ls), run the following command:
 
-**Note:** Python 2.7.9 onwards include ```pip``` by default, so you may have ```pip``` already.
-
-To install mbed-ls from [PyPI](https://pypi.python.org/pypi/mbed-ls) use command:
-```
-$ pip install mbed-ls --upgrade
+```bash
+pip install mbed-ls --upgrade
 ```
 
 ## Installation from Python sources
 
-**Prerequisites:** you need to have [Python 2.7.x](https://www.python.org/download/releases/2.7/) or [Python 3.6.x](https://www.python.org/downloads/release/python-362/) installed on your system.
-
-**Note:** if your OS is Windows, please follow the installation instructions [for the serial port driver](https://developer.mbed.org/handbook/Windows-serial-configuration).
+**Prerequisites:** You need to have [Python 2.7.x](https://www.python.org/download/releases/2.7/) or [Python 3.6.x](https://www.python.org/downloads/release/python-362/) installed on your system.
 
-To install the mbed-ls module:
+**Note:** If your OS is Windows, please follow the installation instructions [for the serial port driver](https://os.mbed.com/docs/latest/tutorials/windows-serial-driver.html).
 
-Clone the mbed-ls repository. The following example uses the GitHub command line tools, but you can do this directly from the website:
+To install the `mbed-ls` module, first clone the `mbed-ls` repository. The following example uses the GitHub command-line tools, but you can do this directly from the website:
 
-```
-$ git clone https://github.com/ARMmbed/mbed-ls.git
+```bash
+git clone https://github.com/ARMmbed/mbed-ls.git
+cd mbed-ls
+python setup.py install
 ```
 
-Change the directory to the mbed-ls repository directory:
+# Use
 
-```
-$ cd mbed-ls
+## Command-line
+
+The command-line tool is available with the command `mbedls`:
```

Comment:
Should we call out the lack of space so that people don't get confused with `mbed ls`?